contrib/completions/bash/kubectl

```diff
@@ -404,6 +404,8 @@ _kubectl_log()
     flags+=("--help")
     flags+=("-h")
     flags+=("--interactive")
+    flags+=("--previous")
```

Comment:
Question: Can this be `last` instead of `previous`? And do we need the `-p` short flag?